frontend/src/scenes/insights/EmptyStates/timeout-state.json

```diff
@@ -562,21 +562,23 @@
                         "properties": [],
                         "filter_test_accounts": false
                     },
-                    "savedFilters": {
-                        "insight": "TRENDS",
-                        "events": [
-                            {
-                                "id": "$pageview",
-                                "name": "$pageview",
-                                "type": "events",
-                                "order": 0
-                            }
-                        ],
-                        "actions": [],
-                        "display": "ActionsLineGraph",
-                        "interval": "day",
-                        "properties": [],
-                        "filter_test_accounts": false
+                    "savedInsight": {
+                        "filters": {
+                            "insight": "TRENDS",
+                            "events": [
+                                {
+                                    "id": "$pageview",
+                                    "name": "$pageview",
+                                    "type": "events",
+                                    "order": 0
+                                }
+                            ],
+                            "actions": [],
+                            "display": "ActionsLineGraph",
+                            "interval": "day",
+                            "properties": [],
+                            "filter_test_accounts": false
+                        }
```

Comment:
Time to start thinking about storybook regression tests?  🤔

Reply:
Shouldn't these JSON dumps be solved by MSW now?